docs/query-builder.txt

```diff
@@ -292,9 +292,37 @@ The inverse of regexp:
 
    User::where('name', 'not regexp', '/.*doe/i')->get();
 
+**ElemMatch**
+
+The :manual:`$elemMatch </reference/operator/query/elemMatch//>` operator
+matches documents that contain an array field with at least one element that
+matches all the specified query criteria.
+
+The following query matches only those documents where the results array
+contains at least one element that is both greater than or equal to 80 and
+is less than 85:
+
+.. code-block:: php
+
+   User::where('results', 'elemMatch', ['gte' => 80, 'lt' => 85])->get();
+
+A sub-query builder can be to apply filter on documents.
```

Comment:
This reads a little rough. Consider:
```suggestion
A closure can be used to create more complex sub-queries.
```

Ping @jmikola in case he has a better suggestion.

Reply:
No objections. Sorry for the delayed response, as I wasn't monitoring mentions in this repo.